src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product-card" data-qa="card">
+      <img 
+        class="product-card__img" 
+        src="./images/imac.jpeg" 
+        alt="imac"
+      >
+      <h1 class="product-card__title">APPLE A1419 iMac 27" Retina 
+        <br> 5K Monoblock (MNED2UA/A)
+      </h1>
+      <p class="product-card__text">Product code: 195434</p>
+      <div class="reviews">
+        <div class="reviews__stars">
```

Comment:
Add the modifier classes for active stars
```suggestion
        <div class="stars stars--4">
```